network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -44,6 +44,38 @@ def __str__(self):
         return self.page.title + '->' + self.blog.title
 
 
+class FeaturedVideoPost(WagtailOrderable, models.Model):
+    page = ParentalKey(
+        'wagtailpages.BlogIndexPage',
+        related_name='featured_video_post',
+    )
+
+    blog_page = models.ForeignKey(
+        'wagtailpages.BlogPage',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+    video_url = models.URLField(
+        help_text='For YouTube: go to your YouTube video and click “Share,” '
+                  'then “Embed,” and then copy and paste the provided URL only. '
+                  'For example: https://www.youtube.com/embed/3FIVXBawyQw '
+                  'For Vimeo: Log into Vimeo using 1Password '
```

Comment:
**Suggestion** I wonder if we could force a line break in the help text between the paragraph on YouTube and the one on Vimeo.